backbone/src/main/res/layout/rsb_view_submitbar.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?><!-- Change to LinearLayout for preview to work -->
 
-<merge xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Please don't remove this, why did you make it a LinearLayout and left it as such?